tests/unit/views/settings/Auth.spec.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable no-unused-expressions */
+import { expect } from 'chai'
+import sinon from 'sinon'
+import Auth from 'corteza-webapp-admin/src/views/Settings/Auth'
+import { stdReject, shallowMount } from 'corteza-webapp-admin/tests/lib/helpers'
+import fp from 'flush-promises'
+
+describe('views/Settings/Auth.vue', () => {
+  afterEach(() => {
+    sinon.restore()
+  })
+
+  let $SystemAPI
+  beforeEach(() => {
+    $SystemAPI = {
+      settingsUpdate: sinon.stub().resolves(),
+      settingsList: sinon.stub().resolves(),
+    }
+
+    sinon.stub(Auth, 'created')
+  })
+
+  const mountEP = (opt) => shallowMount(Auth, {
```

Comment:
This should be `mountAuth`. I usually name these functions like so:

* If it's mounting a component with 1 word (Auth), I use mountAuth
* If it's mounting a component with multiple words (SomeAuth), I shorten with first letters -- mountSA

Reply:
Same applies to all 3 test specs

Reply:
Okey, fixing it.